app/views/layouts/settings.html.slim

```diff
@@ -10,3 +10,5 @@ html
           = yield :sidebar
         .col-sm-9
           = yield
+    footer
+      = yield :google_charts_api
```

Comment:
@yunixon make it `yield :footer`. So there may be any data rendered, not only google chart api.